tests/e2e/README.md

```diff
@@ -252,6 +252,108 @@ Alternatively, set the following environment variables to change these file path
 
 `TCP_ECHO_*` are used in the `dual-stack` test suite, `SLEEP_YAML_PATH` and `HELLOWORLD_YAML_PATH` are used in both `Multicluster` and default test run.
 
+### Using the e2e framework to test your cluster configuration
+The e2e framework can be used to test your cluster configuration. The framework is designed to be flexible and extensible. It is easy to add new test suites and new tests. The idea is to be able to simulate what a real user scenario looks like when using the operator.
+
+To do this, we have a set of test cases under a test group called `smoke`. This test group will run a set of tests that are designed to test the basic functionality of the operator. The tests in this group are designed to be run against a cluster that is already configured and running. The tests will not modify the cluster configuration or the operator configuration. The tests will only check if the operator is working as expected.
+
+Pre-requisites:
+* The operator is already installed and running in the cluster.
+
+To run the test group, you can use the following command:
+
+* Run the following command to run the smoke tests:
+For running on kind:
+```
+$ SKIP_BUILD=true SKIP_DEPLOY=true GINKGO_FLAGS="-v --label-filter=smoke" make test.e2e.kind
+```
+
+For running on OCP:
+```
+$ SKIP_BUILD=true SKIP_DEPLOY=true GINKGO_FLAGS="-v --label-filter=smoke" make test.e2e.ocp
+```
+
+#### Running with specific configuration for the Istio and IstioCNI resource
+It can be cases where you want to run the test with a specific configuration for the Istio and IstioCNI resource to match some cluster specific needs. For this, you can modify the `pkg/istiovalues/vendor_default.yaml`file to default `spec.values` for the Istio and IstioCNI resources. 
+
+This file will be used by the operator to set specific `spec.values` for the Istio and IstioCNI resources. Example:
```

Comment:
we're already explaining this in the vendor_defaults.yaml file, should we just link to that? I'm worried we're duplicating information with the risk of one getting outdated